packages/edit-site/src/components/post-fields/index.js

```diff
@@ -258,11 +283,10 @@ function usePostFields( viewType ) {
 			{
 				label: __( 'Status' ),
 				id: 'status',
-				getValue: ( { item } ) =>
-					STATUSES.find( ( { value } ) => value === item.status )
-						?.label ?? item.status,
+				type: 'text',
 				elements: STATUSES,
 				render: PostStatusField,
+				Edit: 'radio',
```

Comment:
When we edit the status in the editor, we do some extra modifications See https://github.com/WordPress/gutenberg/blob/b12e89732e467e3bb4f8ebcb296ba224be564131/packages/editor/src/components/post-status/index.js#L150-L164

I wonder how we should deal with that.

Reply:
And that's not all of it. We also show the publish date when a user selects `future`. Probably it makes sense to export privately the control we use in the `editor` package and reuse it here?

Reply:
> And that's not all of it. We also show the publish date when a user selects future.

This for me is a weird interactions and I know designers feel like it's a good thing but I have some doubts about it personally :) 

For the `handleStatus` function, we can either absorb it as an API or move it to the `onChange` handler of the `DataForm` instance.

Reply:
Adding or not more components to the status field can be a follow-up. Were we to do that, we could create a custom `Edit` function for `status`.